File: 351004/Dashkevich/Kafka_lab/publisher/src/main/java/com/adashkevich/kafka/lab/dto/response/NewsResponseTo.java
package com.adashkevich.kafka.lab.dto.response;

import java.util.Set;


public class NewsResponseTo {
    public Long id;
    public Long editorId;
    public String title;
    public String content;
    public String created;   // ISO8601
    public String modified;  // ISO8601
    public Set<Long> markerIds;
}

File: 351004/Zalivako/DistributedComputing/src/DiscussionService/DTOs/Responses/NewsResponseTo.cs
namespace DiscussionService.DTOs.Responses
{
    public class NewsResponseTo(
        string title, 
        string content, 
        DateTime createdAt, 
        DateTime modified)
    {
        public long Id { get; set; }

        public long EditorId { get; set; }

        public string Title { get; set; } = title;

        public string Content { get; set; } = content;

        public DateTime CreatedAt { get; set; } = createdAt;

        public DateTime Modified { get; set; } = modified;
    }
}
